### PR TITLE
[DEN-71] Update custom rule set

### PR DESCRIPTION
**Pull request by ssaluja4**

### CODE REVIEW

**Comment by ssaluja4** [comment]
SDK updates : https://github.com/EdgeCast/ec-sdk-go/pull/61

File: ec/resources/waf/custom_rule_set.go [hunk 1]
```diff
@@ -449,8 +449,47 @@ func ResourceCustomRuleSetUpdate(ctx context.Context,
...
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	resp, err := wafService.UpdateCustomRuleSet(accountNumber,
+		customRuleSetID,
+		customRuleSetRequest,
+	)
+	if err != nil {
+		d.SetId("")
```
[the remark this answers is not in the document]
Comment (ssaluja4) [reply ec/resources/waf/custom_rule_set.go]: Removed.

File: ec/resources/waf/custom_rule_set.go [hunk 2]
```diff
@@ -449,8 +449,47 @@ func ResourceCustomRuleSetUpdate(ctx context.Context,
...
+
+	log.Printf("[INFO] Updating WAF Custom Rule Set ID %s for Account >> %s",
+		customRuleSetID,
+		accountNumber,
+	)
+
+	customRuleSetRequest := sdkwaf.UpdateCustomRuleSetRequest{}
+	customRuleSetRequest.Name = d.Get("name").(string)
+
+	directives, err := ExpandDirectives(d.Get("directive"))
```
[the remark this answers is not in the document]
Comment (ssaluja4) [reply ec/resources/waf/custom_rule_set.go]: The expand/flatten methods do not need to be public. Ive updated methods.

[the remark this answers is not in the document]
Comment (ssaluja4) [reply ec/resources/waf/custom_rule_set.go]: Will create a separate branch to clean up the methods in the rest of the project

**Review by changgyu-oh — APPROVED** [review]
looks good!

**Review by wdawg33 — APPROVED** [review]

**Pull request merged**